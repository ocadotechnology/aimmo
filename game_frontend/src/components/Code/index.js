import styled from 'styled-components'

const Code = styled.p`
  font-family: ${props =>
    props.theme.additionalVariables.typography.code.fontFamily};
  color: ${props => props.theme.palette.primary.main};
`
export default Code
