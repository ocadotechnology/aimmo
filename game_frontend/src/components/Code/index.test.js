/* eslint-env jest */
import React from 'react'
import Code from 'components/Code'
import createShallowWithTheme from 'testHelpers/createShallow'

describe('<Code />', () => {
  it('renders correctly', () => {
    const tree = createShallowWithTheme(<Code>Code</Code>)
    expect(tree).toMatchSnapshot()
  })
})
