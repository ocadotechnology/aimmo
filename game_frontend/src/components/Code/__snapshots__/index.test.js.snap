// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Code /> renders correctly 1`] = `
.c0 {
  font-family: 'Source Code Pro',monospace;
  color: #FFFFFF;
}

<p
  className="c0"
>
  Code
</p>
`;
